### PR TITLE
SONAR-23062 Add CI Task that tests Openshift installations

### PR DESCRIPTION
**Pull request by carminevassallo**

### CODE REVIEW

File: charts/sonarqube-dce/ci/openshift-verifier-values.yaml [hunk 1]
```diff
@@ -1,9 +1,12 @@
 OpenShift:
-  enabled: false
+  enabled: true
```
Comment (jCOTINEAU) [inline charts/sonarqube-dce/ci/openshift-verifier-values.yaml:2]: Those values will be used by the chart testing tool by default, hence why the cirrus job for the `chart_testing_on_eks` might be failing as we are using openshift values on EKS

Comment (carminevassallo) [reply charts/sonarqube-dce/ci/openshift-verifier-values.yaml:2]: Fixed!

File: .cirrus/tasks.yml [hunk 2]
```diff
@@ -207,8 +181,39 @@ chart_testing_task:
     - ah lint
   depends_on:
     - chart_static_compatibility_test
-    - verify_openshift_sonarqube
-    - verify_openshift_sonarqube_dce
+
+chart_testing_on_openshift_task:
+  # only_if: $TRIGGER == "VERIFY_OPENSHIFT" || $BRANCH == "master"
```
Comment (jCOTINEAU) [inline .cirrus/tasks.yml:186]: This comment can probably be removed ?

**Review by jCOTINEAU — APPROVED** [review]
lgtm, super cool to see the verifier fully running on OpenShift

File: .cirrus/tasks.yml (hunk 2 above)
Comment (carminevassallo) [reply .cirrus/tasks.yml]: Fixed!

**Pull request merged**